src/App.jsx

```diff
@@ -14,57 +16,57 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
follose common convention about naming setState functions...

Reply:
Still not fixed
<img width="909" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/64750958/100d7665-9327-4e50-9cbc-44e28be356a2">